JASP-Engine/JASP/po/R-nl.po

```diff
@@ -6337,73 +6756,312 @@ msgstr "Posterior Kengetallen Voor Submodel %s"
+
+msgid "The weights (%s) correspond to %s <em>p</em>-values."
+msgstr "De gewichten (%s) corresponderen met de %s <em>p</em>-waarden."
+
+msgid "Estimated studies' effects (%s) correspond to effect size %s."
+msgstr ""
+
+msgid "Effect size (%s)"
+msgstr "Effectgrootte (%s)"
+
+msgid "Heterogeneity (%s)"
+msgstr "Heterogeniteit (%s)"
+
+msgid "Prior Plots"
+msgstr "Prior Grafieken"
+
+msgid "Weight Function"
+msgstr "Gewichtsfunctie"
+
+msgid "Model Preview"
+msgstr "Model Voorvertoning"
+
+msgid ""
+"At least one prior distribution per parameter must be specified (either null "
+"or alternative)."
+msgstr ""
+"Tenminste een a priori distributie per parameter moet gespecificeerd zijn "
+"(in het null of alternatief model)."
+
+msgid "Publication bias"
+msgstr "Publicatieonzuiverheid"
+
+msgid "Models Overview"
+msgstr "Model Overzicht"
+
+msgid "Prior Distribution"
+msgstr "Posterior Verdeling"
+
+msgid "Publication Bias"
+msgstr "Publicatieonzuiverheid"
+
+msgid "The loaded object is not a RoBMA model."
+msgstr "Het ingeladen object is geen RoBMA model."
+
+msgid "Summary"
+msgstr "Samenvatting"
+
+msgid "Exclusion"
+msgstr "Uitsluiting"
+
+msgid "Inclusion"
+msgstr "Inclusie"
+
+msgid "log(BF)"
+msgstr "log(BF)"
+
+msgid ""
+"All models failed to converge. Please, consider inspecting the 'MCMC "
+"diagnostics' and changing the 'Advanced options'."
+msgstr ""
+"Geen model is geconvergeerd. Inspecteer 'MCMC diagnostiek' en verander "
+"de 'Geavanceerde Opties'."
+
+msgid "%i models failed to converge."
+msgstr "%i modellen zijn niet geconvergeerd."
+
+msgid "Model Averaged Estimates"
+msgstr "Model Gemiddelde Schattingen"
+
+msgid "Model Averaged Weights (%s)"
+msgstr "Model Gemiddelde Gewichten (%s)"
+
+msgid "Model Averaged Estimated Studies' Effects (%s)"
+msgstr "Model Gemiddelde Geschatte Onderzoekseffecten"
+
+msgid "Conditional Estimates"
+msgstr "Geconditioneerde Schattingen"
+
+msgid ""
+"Estimates are model averaged over models assuming existence of effect / "
+"heterogeneity."
+msgstr ""
+"De schattingen zijn gemiddeld over modellen waarin het effect bestaat, of waar "
+"heterogeniteit aanwezig is."
+
+msgid "Conditional Weights (%s)"
+msgstr "Geconditioneerde Gewichten %s"
+
+msgid ""
+"Estimated weights (%s) are model averaged over models assuming existence of "
+"publication bias."
+msgstr ""
+"Geschatten gewichten (%s) zijn gemiddeld over modellen waarin "
+"publicatieonzuiverheid aanwezig is."
+
+msgid "Conditional Estimated Studies' Effects (%s)"
+msgstr "Geconditioneerde Geschatte Onderzoekseffecten (%s)"
 
 msgid ""
-"Warning: McDonald's %s if item dropped can only be calculated for three or "
-"more variables."
+"Estimated studies effects (%s) are model averaged over models assuming "
+"existence of effect."
 msgstr ""
-"McDonald's %s Als item wordt weggelaten kan allen worden berekend voor drie "
-"of meer variabelen."
+"Geschatte onderzoekseffecten (%s) zijn gemiddeld over modelleen waarin "
```

Comment:
modellen

Maar "averaged over models assuming existence of effect" is niet hetzelfde als "zijn gemiddeld over modellen waarin het effect bestaat" 

Zoiets?:
"Geschatte onderzoekseffecten (%s) zijn gemiddeld over modellen er van uitgaande dat het effect bestaat."

Reply:
"Geschatte onderzoekseffecten (%s) zijn gemiddeld over die modelleen waarin "
"het effect bestaat."

Reply:
modelleen?

Reply:
IMO zegt de NL zin nu iets anders dan de Engelse.

Reply:
Volgens mij staat het goed in het Nederlands. Sommige modellen hebben het effect en ander modellen niet, je middelt alleen over de modellen waar het effect er in zit. Er zijn veel rare Engelse voetnoten, maar daar snap ik niks, dus schrijf ik maar op wat er hoort te staan.